gstools/field/base.py

```diff
@@ -146,6 +154,58 @@ def mesh(
                 mesh.point_data[name] = field
         return out
 
+    def _pre_pos(self, pos, mesh_type="unstructured", make_unstruct=False):
+        """
+        Preprocessing positions and mesh_type.
+
+        Parameters
+        ----------
+        pos : :any:`iterable`
+            the position tuple, containing main direction and transversal
+            directions
+        mesh_type : :class:`str`
+            'structured' / 'unstructured'
+        make_unstruct: :class:`bool`
+            State if mesh_type should be made unstructured.
+
+        Returns
+        -------
+        x : :class:`numpy.ndarray`
+            first components of unrotated and isotropic position vectors
+        y : :class:`numpy.ndarray` or None
+            analog to x
+        z : :class:`numpy.ndarray` or None
+            analog to x
+        pos : :class:`tuple` of :class:`numpy.ndarray`
+            the normalized position tuple
+        mesh_type_gen : :class:`str`
+            'structured' / 'unstructured' for the generator
+        mesh_type_changed : :class:`bool`
+            State if the mesh_type was changed.
+        axis_lens : :class:`tuple` or :any:`None`
+            axis lenghts of the structured mesh if mesh type was changed.
+        """
+        x, y, z = pos2xyz(pos, max_dim=self.model.dim)
+        pos = xyz2pos(x, y, z)
+        mesh_type_gen = mesh_type
+        # format the positional arguments of the mesh
+        check_mesh(self.model.dim, x, y, z, mesh_type)
+        mesh_type_changed = False
+        axis_lens = None
+        if (
+            self.model.do_rotation or make_unstruct
+        ) and mesh_type == "structured":
```

Comment:
This is very unreadable. I suggest a line break before or after `and`.

Reply:
The holy black said it is ok so. Ahmen.